docs/pages/machine-workload-identity/machine-id/deployment/bitbucket.mdx

```diff
@@ -120,12 +120,11 @@ pipelines:
       - step:
           oidc: true
           script:
-            # Download and extract Teleport
-            - wget https://cdn.teleport.dev/teleport-v(=teleport.version=)-linux-amd64-bin.tar.gz
-            - tar -xvf teleport-v(=teleport.version=)-linux-amd64-bin.tar.gz
+            # Install Teleport
+            - curl "https://example.teleport.sh:443/scripts/install.sh" | bash
```

Comment:
Are there instructions to replace the Proxy Service URL here?

Reply:
It's not essential, but should we use `Var` components for Proxy Service address values that the user needs to replace?